Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -198,6 +198,9 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
 
 		protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
+			if (_disposed)
```

Comment:
OK, we've found the following to be effective in creating a repro for these **ObjectDisposedException** problems:

1. Push/pop the navigation stack i.e. Push the demo page onto the stack and then pop it off again.
2. Simultaneously when pushing/popping, manually fire a GC collect to simulate memory pressure and run the GC sweep.
3. Use DataTriggers or bindings on a XAML view with a view-model that uses INotifyPropertyChanged. You need to get these bindings/triggers firing to see the problem.

Doing all three of these things, we have been able to repro ObjectDisposedException crashes  today on `Label` and `ProgressBar`. See our repro project here for an example:
https://bugzilla.xamarin.com/show_bug.cgi?id=57910